community/samples/serving/helloworld-python-bentoml/README.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Hello World - Python BentoML"
+linkTitle: "Python Bentoml"
+weight: 1
+type: "docs"
+---
+
+A simple machine learning model with API serving that is written in python and
+using [BentoML](https://github.com/bentoml/BentoML). BentoML is an open source
+framework for high performance ML model serving, which supports all major machine
+learning frameworks including Keras, Tensorflow, PyTorch, Fast.ai, XGBoost and etc.
+
+This sample will walk you through the steps of creating and deploying a machine learning
+model using python. It will use BentoML to package a classifier model trained
+on the Iris dataset. Afterward, it will create a container image and
+deploy the image to Knative.
+
+Knative deployment guide with BentoML is also available in the
+[BentoML documentation](https://docs.bentoml.org/en/latest/deployment/knative.html)
+
+## Before you begin
+
+- A Kubernetes cluster with Knative installed. Follow the
+  [installation instructions](../../../../docs/install/README.md) if you need to
+  create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured. Docker Hub will be used for a container registry).
+- Python 3.6 or above installed and running on your local machine.
+  - `scikit-learn` and `bentoml` packages, run command:
+
+    ```shell
+    pip install scikit-learn
+    pip install bentoml
+    ```
+
+## Recreating sample code
+
+1. BentoML creates a model API serving, via prediction service abstraction. This code
+  defines a prediction service that requires a scikit-learn model. It asks BentoML to
+  figure out the required pip dependencies, also defined an API, which is the entry
+  point for accessing this machine learning service. Save the
+  following code into a file named `iris_classifier.py`:
+
+    ```python
```

Comment:
I'd suggest putting these files into the same directory, and then including them here with the following shortcode (will work on the website which is rendered by hugo, but not in github markdown): `{{% readfile file="iris_classifier.py" %}}`

That way, people can also check out the files directly from GitHub if they don't want to copy-paste.

Reply:
Great suggestion, I remove the code blocks into actual files.